### ISSUE TITLE
Unable to delta_scan for delta lake on GCS

### ISSUE THREAD

**Issue by yohei1126** [duckdb/pg_duckdb#492]
### What happens?

delta_scan tried to access S3 bucket and failed even if I give GCS bucket file path.

### To Reproduce

Insert GCS HMAC key https://duckdb.org/docs/guides/network_cloud_storage/gcs_import.html
```
$ INSERT INTO duckdb.secrets (type, key_id, secret)
  VALUES ('GCS', 'xx'x, 'yyy');
INSERT 0 1
```

```
# SELECT * FROM  delta_scan ('gs://databricks-xxx-unitycatalog/yyy') AS (color text,  price numeric);
ERROR:  (PGDuckDB/CreatePlan) Prepared query returned an error: 'IO Error: Hit DeltaKernel FFI error 
(from: While trying to read from delta table: 'gs://databricks-xxx-unitycatalog/yyy/'): Hit error: 8 (ObjectStoreError)
with message (Error interacting with object store: Generic S3 error: Error after 10 retries in 2.43329041s, 
max_retries:10, retry_timeout:180s, source:error sending request for url (https://s3..amazonaws.com/databricks-xxxx-unitycatalog/yyy/_delta_log/_last_checkpoint))

```

### OS:

MacOS

### pg_duckdb Version (if built from source use commit hash):

0.2.0

### Postgres Version (if built from source use commit hash):

16

### Full Name:

Yohei

### Affiliation:

Onishi

### What is the latest build you tested with? If possible, we recommend testing with the latest nightly build.

I have tested with a stable release

### Did you include all relevant data sets for reproducing the issue?

No - I cannot share the data sets because they are confidential

### Did you include all code required to reproduce the issue?

- [X] Yes, I have

### Did you include all relevant configuration (e.g., CPU architecture, Linux distribution) to reproduce the issue?

- [X] Yes, I have

**Comment by Y--** [comment]
Hi @yohei1126! Thanks for the report. Just to double check, does the same work on vanilla DuckDB? (using the CLI for example?) Thanks!

**Comment by yohei1126** [comment]
@Y--  Hi, delta_scan work on vanilla DuckDB
https://duckdb.org/docs/extensions/delta.html

**Comment by JelteF** [comment]
@yohei1126 but did you try this same exact query in plain duckdb (with the same dataset)? Does it fail there too or does it succeed? The reason we ask is because the error does not seem pg_duckdb specific.

**Comment by yohei1126** [comment]
yes, I have already tested this query in plain duckedb. It works. 
```
INSTALL delta;
LOAD delta;
CREATE OR REPLACE SECRET (
    TYPE GCS,
    KEY_ID 'xxxx'
    SECRET 'yyy'
);
SELECT *FROM delta_scan('gs://bucket_name/xxx/yyy/zzz/')
```

bucket_name/xxx/yyy/zzz/ is a path to the directory where I have delta lake files

**Comment by JelteF** [comment]
Can you share a public bucket that exposes this problem? It's hard to help debug this without a reproducible example.

**Comment by JelteF** [comment]
Closing this due to lack of info to reproduce the issue. Feel free to create a new issue/reopen this one if you have more info.

**Comment by yohei1126** [comment]
Thanks. I will re-open this issue until the issue is resolved.
